modules/ROOT/pages/release-notes.adoc

```diff
@@ -92,9 +92,9 @@ The Tomcat bundle does not log anymore in the console but only in `bonita.log` f
 
 === Enterprise Docker image 
 
-Docker image is now focused on the RUN phase, environment preparation will be handled apart (as on-premise installations) so docker image does not create anymore the database schema and the associated user. Still, the database must be created before the docker image starts. We provide pre-configured database images with included schema and users on Docker Hub. 
+Docker image is now focused on the RUN phase, environment preparation will be handled apart (as on-premise installations) so docker image does not create anymore the database schema and the associated user. Still, the database must be created before the docker image starts. We provide pre-configured database images with included schema and users on xref:https://hub.docker.com/u/bonitasoft[Docker Hub].
```

Comment:
here, you should remove the `xref:` part and use the same syntax for url as you did in `modules/runtime/pages/bonita-docker-installation.adoc`
See https://docs.asciidoctor.org/asciidoc/latest/macros/url-macro/ for more details

Reply:
ok, thx